### ISSUE TITLE
Create release step should have a concurrency lock

### ISSUE THREAD

**Issue by robdimsdale** [paketo-buildpacks/jammy-full-stack#6]
Ultimately this will probably need to be in https://github.com/paketo-buildpacks/github-config but I'm capturing it here because this is the affected repo.

Currently we have multiple jobs running in parallel. This is happening because the timer is faster than the job. See below:

<img width="847" alt="Screen Shot 2022-07-26 at 2 19 31 PM" src="https://user-images.githubusercontent.com/7230694/181082041-7bc9be1d-a488-4dd5-b1b0-63c4c1cf11e1.png">

I think we could also reduce the frequency, but we should ensure that only one build runs at a time.

**Comment by robdimsdale** [comment]
Oh, actually, it looks like we canceled the jobs when the newer one came in. I think that is probably not desirable behavior as it means we could potentially fail to create releases from specific commits - e.g. if we merge PRs too quickly.

**Comment by robdimsdale** [comment]
Upon further investigation this appears to be behaving properly. Scheduled workflows have lower priority than workflows triggered by commits, so we should successfully run the workflow for each commit that comes in.

We should validate that we successfully build a release when there is a new patched USN - and that we don't cancel each in flight job when the next USN poll triggers.

Closing this.
